### ISSUE TITLE
postcss-loader failure in parsing .sccs file comment

### ISSUE THREAD

**Issue by st-clair-clarke** [ngneat/tailwind#4]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request
[ ] Other... Please describe:
</code></pre>

## Current behavior

I added tailwind to my nx app using the command nx add @ngneat/tailwind.
The installation was fine.

## Expected behavior

My application should be served normally as before the installation.

## Minimal reproduction of the problem with instructions

Serving my app gives the following error

```
ERROR in J:/@workspace/nrwl/pim/node_modules/@swimlane/ngx-datatable/themes/material.scss (J:/@workspace/nrwl/pim/node_modules/css-loader/dist/cjs.js??ref--14-1!J:/@workspace/nrwl/pim/node_modules/postcss-loader/src??embedded!J:/@workspace/nrwl/pim/node_modules/resolve-url-loader??ref--14-3!J:/@workspace/nrwl/pim/node_modules/sass-loader/dist/cjs.js??ref--14-4!J:/@workspace/nrwl/pim/node_modules/postcss-loader/src??ref--18!J:/@workspace/nrwl/pim/node_modules/@swimlane/ngx-datatable/themes/material.scss)
Module build failed (from J:/@workspace/nrwl/pim/node_modules/postcss-loader/src/index.js):
SyntaxError

(19:4) Unknown word

  17 | */
  18 |
> 19 | // common datatable colors
     |    ^
  20 | $ngx-datatable-background: #fff !default;
  21 | $ngx-datatable-box-shadow: 0 5px 5px -3px rgba(0, 0, 0, 0.2), 0 8px 10px 1px rgba(0, 0, 0, 0.14),


ERROR in Module build failed (from J:/@workspace/nrwl/pim/node_modules/postcss-loader/src/index.js):
SyntaxError

(26:13) Unknown word

  24 |
  25 |     ::ng-deep .dx-treeview {
> 26 |       // ## Long text positioning
     |             ^
  27 |       white-space: nowrap;
  28 |       // ##


ERROR in ./src/styles.scss (J:/@workspace/nrwl/pim/node_modules/css-loader/dist/cjs.js??ref--14-1!J:/@workspace/nrwl/pim/node_modules/postcss-loader/src??embedded!J:/@workspace/nrwl/pim/node_modules/resolve-url-loader??ref--14-3!J:/@workspace/nrwl/pim/node_modules/sass-loader/dist/cjs.js??ref--14-4!J:/@workspace/nrwl/pim/node_modules/postcss-loader/src??ref--18!./src/styles.scss)
Module build failed (from J:/@workspace/nrwl/pim/node_modules/postcss-loader/src/index.js):
SyntaxError

(4:1) Unknown word

  2 | /* You can add global styles to this file, and also import other style files */
  3 |
> 4 | //////////////////////////////// TAILWIND CSS //////////////////////////////////
    | ^
  5 | @import 'node_modules/tailwindcss/base';
  6 | @import 'node_modules/tailwindcss/components';
```

Apparently comments within .scss files are not parsed.

For bug reports please provide the _STEPS TO REPRODUCE_ and if possible a _MINIMAL DEMO_ of the problem

## Environment

<pre><code>
Angular version: 10.0.8

Browser:
- [x ] Chrome (desktop) version XX
- [ ] Chrome (Android) version XX
- [ ] Chrome (iOS) version XX
- [ ] Firefox version XX
- [ ] Safari (desktop) version XX
- [ ] Safari (iOS) version XX
- [ ] IE version XX
- [ ] Edge version XX
 
For Tooling issues:
- Node version: 14.8.0
- Platform:  Windows 10

Others:

</code></pre>

**Comment by nartc** [comment]
Hi @st-clair-clarke, I can't seem to reproduce the issue. Can you please share a sample repo? or can you share the `webpack.config.js`?

**Comment by st-clair-clarke** [comment]
Thanks for the reply.
Somehow, all my style paths seem to not work any more with errors such as:

```
Error: Failed to find '../../../../../../../apps/medsoft/src/themes/generated/variables.base'
```

The path is correct, however. Only after installation I am getting the errors

Here are the file requested:

## webpack.config.js
```
const merge = require('webpack-merge');

module.exports = (config) => {
  const isProd = config.mode === "production";
  const tailwindConfig = require("./tailwind.config.js")(isProd);

  return merge(config, {
    module: {
      rules: [
        {
          test: /\.scss$/,
          loader: 'postcss-loader',
          options: {
            postcssOptions: {
              ident: 'postcss',
              syntax: 'postcss-scss',
              plugins: [
                require('postcss-import'),
                require('tailwindcss')(tailwindConfig),
                require('autoprefixer'),
              ]
            }
          }
        }
      ]
    }
  });
};
```

## tailwind.config.js

```
module.exports = (isProd) => ({
    prefix: '',
    future: {
      removeDeprecatedGapUtilities: true,
      purgeLayersByDefault: true
    },
    purge: {
      enabled: isProd,
      content: ['**/*.html', '**/*.ts']
    },
    theme: {}
});


```
```

**Comment by nartc** [comment]
@st-clair-clarke that's `postcss-import` limitation. You'd need to use relative path and file extension when you use regular SCSS import when `postcss-import` is setup. Like the following:

```scss
@import 'relative-path/to/variable.scss';
```

Give this a try.

**Comment by st-clair-clarke** [comment]
Are all styles now relative to the root directory?

**Comment by nartc** [comment]
@st-clair-clarke to where you import it

**Comment by nartc** [comment]
@st-clair-clarke Styles in angular.json would work as normal. But you’d need to make sure of the order of your stylesheets. If you don’t have overriding styles that are concerned with the order of stylesheets, styles in angular.json would be a tad cleaner

**Comment by nartc** [comment]
Close due to inactivity. Feel free to open a new issue if you're still running into problem :)

**Comment by zetsnotdead** [comment]
I've had the same issue and the fix was updating `postcss-scss` from version `2.x` to `3.x`
